### PR TITLE
Added new MCQ to security quiz file with relevant freeCodeCamp link

### PR DESCRIPTION
**Pull request by TannisthaBiswas**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

File: src/data/security-quiz.ts [hunk 1]
```diff
@@ -307,6 +307,18 @@ const securityQuiz = [
       "Partial Homomorphic Encryption (PHE) allows only one operation to be performed on the ciphertext an infinite number of times. This operation can be only addition or only multiplication.",
     Link: "https://www.freecodecamp.org/news/introduction-to-homomorphic-encryption/",
   },
+
+  {
+    Question:
+      "What is the full form os SSL?",
```
Comment (jdwilkin4) [inline src/data/security-quiz.ts:313]:
```suggestion
      "What does SSL stand for?",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by TannisthaBiswas** [comment]
Changes incorporated. Hopefully you can merge now. 😊

**Review by jdwilkin4 — APPROVED** [review]
Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**